### ISSUE TITLE
Slash commands broken when opening PR for non-master branch

### ISSUE THREAD

**Issue by wildlifehexagon** [peter-evans/slash-command-dispatch#76]
I'm attempting to run this action locally using the same example from the documentation. However, when I execute a slash command inside a pull request that is merging `feature/xyz` into `develop` (i.e. a non-master branch), the dispatch action breaks entirely.

Here's my dispatch workflow

```yaml
name: Slash Command Dispatch
on:
  issue_comment:
    types: [created]
jobs:
  dispatch:
    runs-on: ubuntu-latest
    steps:
      - name: Dispatch slash command (example)
        uses: peter-evans/slash-command-dispatch@v2
        with:
          token: ${{ secrets.REPO_ACCESS_TOKEN }}
          reaction-token: ${{ secrets.REPO_ACCESS_TOKEN }}
          issue-type: pull-request
          commands: example
```

and the example command workflow

```yaml
name: Slash Command Processor (Example)
on:
  repository_dispatch:
    types: [example-command]
jobs:
  example:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
        with:
          repository: ${{ github.event.client_payload.pull_request.head.repo.full_name }}
          ref: ${{ github.event.client_payload.pull_request.head.ref }}
      - name: Add reaction
        uses: peter-evans/create-or-update-comment@v1
        with:
          token: ${{ secrets.REPO_ACCESS_TOKEN }}
          comment-id: ${{ github.event.client_payload.github.payload.comment.id }}
          reaction-type: hooray
```

The error I get from the first action (after leaving a slash command comment in the PR) is this:

```
Using configuration from yaml inputs.
Command 'example' to be dispatched.
##[error]Cannot read property 'slice' of null
```

Is it possible to use this action when merging into a non-master branch?

**Comment by peter-evans** [comment]
Hi @wildlifehexagon 

I think you might have stumbled across a bug, but I can't seem to reproduce it. Does the pull request `body` (the main content section of the pull request) have any content? Or is it empty?

This has nothing to do with the fact that the pull request is merging into a non-master branch. It should work fine.

**Comment by wildlifehexagon** [comment]
Hi, thanks for the quick reply. The pull request body is empty. What's weird is that doing a pull request into `master` with these exact same workflows works perfectly. That's why I was thinking it might have something to do with the repo's default branch being `develop`. Here's an example of a PR I was playing around with earlier:

https://github.com/dictybase-playground/github-actions-experiments/pull/13

**Comment by peter-evans** [comment]
Can you try putting some content in the pull request body and make another slash command comment. See if that fixes it.
If it does, it's a bug which is easily fixed.

**Comment by wildlifehexagon** [comment]
That did it, thanks!

**Comment by peter-evans** [comment]
Great! Thanks for confirming.
I'll roll out a fix for this today.

**Comment by peter-evans** [comment]
Fixed in `v2.1.1` / `v2`.
Thanks for reporting!
